crm_claim_rma/views/crm_claim.xml

```diff
@@ -0,0 +1,291 @@
+            res_model="stock.picking"
+            src_model="crm.claim"/>
+
+
+    <record model="ir.ui.view" id="crm_claim_rma_form_view">
+        <field name="name">CRM - Claim product return Form</field>
+        <field name="model">crm.claim</field>
+        <field name="inherit_id" ref="crm_claim.crm_case_claims_form_view"/>
+        <field name="arch" type="xml">
+            <!-- Header/workflow Buttons -->
+            <xpath expr="//field[@name='stage_id']" position="before">
+                <button name="%(action_claim_picking_in)d"
+                        string="New Products Return"
+                        class="oe_inline"
+                        type="action" target="new"
+                        context="{'warehouse_id': warehouse_id,
+                            'partner_id': partner_id}"/>
+
+                <button name="%(action_claim_picking_out)d"
+                        class="oe_inline"
+                        string="New Delivery"
+                        type="action" target="new"
+                        context="{'warehouse_id': warehouse_id,
+                            'partner_id': partner_id}"/>
+
+                <button name="%(account.action_account_invoice_refund)d"
+                        class="oe_inline"
+                        type='action' string='New Refund'
+                        context="{
+                            'invoice_ids': [invoice_id],
+                            'claim_line_ids': claim_line_ids,
+                            'description': name,
+                            'claim_id': id,
+                            }"/>
+            </xpath>
+            <xpath expr="//field[@name='team_id']/parent::group" position="replace">
+                <group colspan="4" col="4" groups="base.group_user">
+                    <field name="warehouse_id"/>
+                    <field name="date"/>
+                    <field name="user_id"
+                           context="{'default_groups_ref': ['base.group_user', 'base.group_partner_manager', 'base.group_sale_salesman_all_leads']}"/>
+                    <field name="priority" widget="priority"/>
+                    <field name="date_deadline"/>
+                    <field name="team_id"/>
+                </group>
+            </xpath>
+
+            <xpath expr="//field[@name='partner_id']" position="attributes">
+                <attribute name="required">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='partner_phone']" position="attributes">
+                <attribute name="required">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='email_from']" position="attributes">
+                <attribute name="required">1</attribute>
+            </xpath>
+
+            <!-- Smart buttons -->
+            <xpath expr="//field[@name='name']/parent::group" position="after">
+                <div class="oe_right oe_button_box" name="buttons">
```

Comment:
You can remove oe_right and s/buttons/button_box in name (it will use the responsive design putting buttons in a More drop down menu)